install/ersatztv-install.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/ErsatzTV/ErsatzTV
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies (Patience)"
+$STD apt-get install -y --no-install-recommends \
+  ffmpeg \
+  build-essential \
+  unzip \
+  pkg-config \
+  curl \
+  sudo \
+  git \
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+
+msg_info "Installing ErsatzTV (Patience)" 
+cd /opt
+RELEASE=$(curl -s https://api.github.com/repos/ErsatzTV/ErsatzTV/releases | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+LATEST_RELEASE=$(echo $RELEASE | awk '{print $1}')
+echo $LATEST_RELEASE
+wget -q --no-check-certificate "https://github.com/ErsatzTV/ErsatzTV/releases/download/${LATEST_RELEASE}/ErsatzTV-${LATEST_RELEASE}-linux-x64.tar.gz"
```

Comment:
why not make it simple?
```
RELEASE=$(curl -s https://api.github.com/repos/ErsatzTV/ErsatzTV/releases | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }' | head -n 1)
wget -q https://github.com/ErsatzTV/ErsatzTV/releases/download/${RELEASE}/ErsatzTV-${RELEASE}-linux-x64.tar.gz
```

Reply:
done, i didn't even know about the head -n 1 variable. Of course, this makes everything much more pleasant. Thanks for the feedback